src/server_manager/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "outline-manager",
   "productName": "Outline Manager",
-  "version": "1.11.0",
+  "version": "1.12.0",
```

Comment:
Is `1.11.1` better since we did not introduce any new features?

Reply:
The last change was a minor bump and we're reverting that, so my logic here was that the revert of a minor patch is also a minor patch.